com.unity.render-pipelines.universal/Runtime/Passes/DrawSkyboxPass.cs

```diff
@@ -55,7 +57,20 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             else
 #endif
             {
-                context.DrawSkybox(renderingData.cameraData.camera);
+	            var sceneOverrideMode = DebugDisplaySettings.Instance.renderingSettings.sceneOverrides;
```

Comment:
The if above handles XR case. I don't exactly remember why we need to replace the skybox material for the overdraw mode however whatever we do we have to make sure to work on XR as well.

Reply:
This goes for the PR as a whole - I haven't tested it in XR, that needs doing.

Reply:
XR codepath is now handled here. XR testing should happen with the follow-up PR.